src/packages.builds

```diff
@@ -53,6 +53,18 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipNativePackageBuild)' != 'true'" >
```

Comment:
I received the opposite feedback from @ericstj on the `release/1.0.0` PR:

https://github.com/dotnet/corefx/pull/18365#discussion_r111622930